lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1895,8 +2010,29 @@ def db_connect_postgresql(*args)
       opts['host'] = '127.0.0.1'
     end
 
-    if (not framework.db.connect(opts))
-      raise RuntimeError.new("Failed to connect to the database: #{framework.db.error}")
+    if framework.db.connect(opts)
+      print_line "Connected to Postgres data service: #{info[:host]}/#{info[:name]}"
+    else
+      raise RuntimeError.new("Failed to connect to the data service: #{framework.db.error}")
+    end
+  end
+
+  def db_connect_http(opts)
+    # local database is required to use Mdm objects
+    unless framework.db.active
+      print_error("No local database connected. Please connect to a local database before connecting to a remote data service.")
+      return
+    end
+
+    uri = db_parse_db_uri_http(opts[:url])
+
+    remote_data_service = Metasploit::Framework::DataService::RemoteHTTPDataService.new(uri.to_s, opts)
+    begin
+      framework.db.register_data_service(remote_data_service)
+      print_line "Connected to remote data service: #{remote_data_service.name}"
```

Comment:
Minor: consider changing to "Connected to data service: ...". Drop remote as it could be a local data service.

Reply:
This method is specific to HTTP connections, so it will always be remote. There is a separate line in the Postgres connect method that specifies it's a Postgres service.

Reply:
I understand that it is a `RemoteHTTPDataService`, however, I was thinking about it from the user perspective. The HTTP service could be on the local system or a remote system.

Reply:
I could change it to HTTP data service. I think it mostly comes down to how we want to handle naming of these things going forward. That's a bigger conversation as there are already a lot of confusing/inconsistent wording choices across all of this (e.g. `db_connect` but you're connecting to an API, not a database). We may want to talk to Thao and see about making this more consistent.

Reply:
That sounds good to me. I agree there is a greater conversation necessary in this space, and probably even more we could do, for example, refactoring `RemoteHTTPDataService` to `HTTPDataService`.